### ISSUE TITLE
sound volume increase between songs in local mode

### ISSUE THREAD

**Issue by stef64** [xbmc/chorus2#235]
Hello,

I noticed that playing albums locally from my browser (any browser) I have to reduce the volume after each songs.

My Kodi image resides on a different machine.

I have to say this is the only hick that I have found, Great work !  Thank you

UPDATE : tested on Firefox 51.0.1: Ubuntu 16.10 seams that the volume in chorus2 affect firefox volume not system volume. It bring volume to the max.
Youtube is using firefox volume but do not bring the volume to max for the following video. 
Knowing that I adjusted my sound accordingly, it is an issue but an easy work around.

**Comment by jez500** [comment]
I can't reproduce this. Does the volume slider actually move or does it just sound louder?

**Comment by stef64** [comment]
Thank you for the reply.
I just tested it on my android and it was all good but since my original issue wasn't on android I tried again on my Ubuntu 16.10. Using Firefox the result was impeccable.
Then I used Chrome and I experienced the same issues as before. ( I will check that issue later )
Nevertheless I tried to install Chromium and it's working great.

So maybe some updates fixed that... don't know. I'll re-install Chrome at a later time and I'll see.

During the WE I was looking around for ways to play all my movies from the browser and some like HEVC are playing sound but without video. From VLC it works fine using nfs to the kodi machine.
Since I have find from you it is not possible to play all formats until kodi allow transcoding I tried to used the bottom left arrow that allow me to switch to VLC but without success.
I enjoy your web Interface and I wonder if some sort of "grab" - copy-paste - drag-drop - from it to VLC client would allow me to view my movies using the nfs link. (I presume chorus2 knows my movies path and I can select nfs links as a choice - for others can be smb)
An extra choice in your 3dots menu can for say copy the nfs links to the clipboard...
It is not perfect and it's a work around until kodi allow transcoding....
